### PR TITLE
Adding README to spring-boot-starter

### LINKED ISSUE #17123: Add README to spring-boot-starter-camunda-sdk
**Description**

Add README to spring-boot-starter-camunda-sdk:

- Copy README file from the community project
- Adjust if needed

### PR DESCRIPTION
**Pull request by ana-vinogradova-camunda**
## Description

Adding README version from the community project. I am not adding the latest version of README file because the community project was updated after we migrated it to Zeebe.
Links to Operate, Tasklist etc. were removed. The section related to ZPT was also removed as this was not a part of the scope.

## Related issues

closes #17123

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

File: spring-boot-starter-camunda-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,574 @@
...
+|----------------------|-------|-----------------|-----------------------------|---------------------------------|
+| >= 8.4.0             | >= 17 | 8.4.0           | 3.2.0                       | >= 2.7.x, 3.x.x                 |
+| >= 8.3.4             | >= 17 | 8.3.4           | 3.2.0                       | >= 2.7.x, 3.x.x                 |
+| >= 8.3.0             | >= 17 | 8.3.1           | 2.7.7                       | >= 2.7.x, 3.x.x                 |
+| >= 8.3.0             | >= 8  | 8.3.1           | 2.7.7                       | >= 2.7.x                        |
+| >= 8.2.4             | >= 17 | 8.2.4           | 2.7.7                       | >= 2.7.x, 3.x.x                 |
+| >= 8.2.4             | >= 8  | 8.2.4           | 2.7.7                       | >= 2.7.x                        |
+| >= 8.1.15            | >= 17 | 8.1.x           | 2.7.7                       | >= 2.7.6, 3.x.x                 |
+| >= 8.1.15            | >= 8  | 8.1.x           | 2.7.7                       | >= 2.7.6                        |
+| <= 8.1.14            | >= 8  | 8.1.x           | 2.7.5                       | = 2.7.x                         |
```
Comment (megglos) [inline spring-boot-starter-camunda-sdk/README.md:44]: 🔧 let's remove the history and start with 8.5.0

**Review by megglos — COMMENTED** [review]
Thank you for stripping the readme down and preparing it for integration, I went ahead and revised every section, fixing links and also revising some general content parts.

As we may even go ahead now and integrate this content into docs.camunda.io right away I would use this as a handover draft for @christinaausley to take over.

Many thanks to @christinaausley for offering her help in @akeller absence to go ahead with the integration into the docs.

**Comment by christinaausley** [comment]
This is documented and revised in https://github.com/camunda/camunda-docs/pull/3552 👍

**Comment by megglos** [comment]
closing in favor of https://github.com/camunda/camunda-docs/pull/3552 thanks @christinaausley for integrating this into the docs right away 🚀

**Pull request closed without merging**